README.md

```diff
@@ -64,54 +38,17 @@ After you're finished please remove all the comments and instructions!
 
 > Describe your project in 1 or 2 sentences.
 
-**[your_project__name]** is a...
```

Comment:
- [ ] Please note that this selected inline is not where I am requesting changes but I cant use inline for that part. Above this selected inline you have `# 📖 [your_project_name] <a name="about-project"></a>` and make sure that you replace  `your_project_name` this your project name that is `Hello Microverse` :smile: